modules/gdscript/gdscript.cpp

```diff
@@ -39,6 +39,9 @@
 #include "core/os/file_access.h"
 #include "core/os/os.h"
 #include "core/project_settings.h"
+#include "editor/doc_tools.h"
+#include "editor/editor_help.h"
+#include "editor/plugins/script_editor_plugin.h"
```

Comment:
I think there's still an encapsulation issue here, `gdscript.cpp` shouldn't depend on editor code.
Editor-specific GDScript features are in `modules/gdscript/editor/`.

Reply:
The editor-specific functions can be implemented in a new file under `editor` (maybe as `gdscript_doc.cpp`). Or you can add to `gdscript_editor.cpp` though that one is a bit cluttered already.

Reply:
That's the only thing left to rework IMO, then we can merge. If you're not sure how to do it, maybe @vnen can help.

Reply:
done! I was using script editor in gdscript to update the docs, where I should've used the `Ref<Script>` interface from the script editor.
@akien-mga  @vnen

Reply:
Awesome! :)

Reply:
save as... didn't update the helper, and I've just fixed it. if there any edge cases let me know